Compile the serial ABACUS failed with RAPID_JSON=ON

### Describe the bug

I try to compile ABACUS with option `-DENABLE_RAPIDJSON=ON -DENABLE_MPI=OFF`, and it is failed with error
```
/abacus/source/module_io/json_output/general_info.cpp:42:6: error: #elif with no expression
   42 | #elif
```

### Expected behavior

_No response_

### To Reproduce

_No response_

### Environment

_No response_

### Additional Context

_No response_

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

--- a/source/module_io/json_output/general_info.cpp
+++ b/source/module_io/json_output/general_info.cpp
@@ -6,12 +6,12 @@
 #include "module_io/input.h"
 #include "version.h"
 
-//Add json objects to gener_info
+// Add json objects to gener_info
 namespace Json
 {
 
 #ifdef __RAPIDJSON
-void gen_general_info(Input *input)
+void gen_general_info(Input* input)
 {
 
 #ifdef VERSION
@@ -32,7 +32,7 @@ void gen_general_info(Input *input)
     convert_time(start_time, start_time_str);
 
     // end_time
-    std::time_t time_now = std::time(NULL);
+    std::time_t time_now = std::time(nullptr);
     std::string end_time_str;
     convert_time(time_now, end_time_str);
 
@@ -44,17 +44,17 @@ void gen_general_info(Input *input)
     int omp_num = 1;
 #endif
 
-    AbacusJson::add_json({"general_info", "version"}, version,false);
-    AbacusJson::add_json({"general_info", "commit"}, commit,false);
-    AbacusJson::add_json({"general_info", "device"}, input->device,false);
-    AbacusJson::add_json({"general_info", "mpi_num"}, mpi_num,false);
-    AbacusJson::add_json({"general_info", "omp_num"}, omp_num,false);
-    AbacusJson::add_json({"general_info", "pseudo_dir"}, input->pseudo_dir,false);
-    AbacusJson::add_json({"general_info", "orbital_dir"}, input->orbital_dir,false);
-    AbacusJson::add_json({"general_info", "stru_file"}, input->stru_file,false);
-    AbacusJson::add_json({"general_info", "kpt_file"}, input->kpoint_file,false);
-    AbacusJson::add_json({"general_info", "start_time"}, start_time_str,false);
-    AbacusJson::add_json({"general_info", "end_time"}, end_time_str,false);
+    AbacusJson::add_json({"general_info", "version"}, version, false);
+    AbacusJson::add_json({"general_info", "commit"}, commit, false);
+    AbacusJson::add_json({"general_info", "device"}, input->device, false);
+    AbacusJson::add_json({"general_info", "mpi_num"}, mpi_num, false);
+    AbacusJson::add_json({"general_info", "omp_num"}, omp_num, false);
+    AbacusJson::add_json({"general_info", "pseudo_dir"}, input->pseudo_dir, false);
+    AbacusJson::add_json({"general_info", "orbital_dir"}, input->orbital_dir, false);
+    AbacusJson::add_json({"general_info", "stru_file"}, input->stru_file, false);
+    AbacusJson::add_json({"general_info", "kpt_file"}, input->kpoint_file, false);
+    AbacusJson::add_json({"general_info", "start_time"}, start_time_str, false);
+    AbacusJson::add_json({"general_info", "end_time"}, end_time_str, false);
 
     // AbacusJson::add_Json(version,false,"general_info", "version");
     // AbacusJson::add_Json(commit,false,"general_info", "commit");

--- a/source/module_io/json_output/general_info.cpp
+++ b/source/module_io/json_output/general_info.cpp
@@ -39,7 +39,7 @@ void gen_general_info(Input *input)
 #ifdef __MPI
     int mpi_num = Parallel_Global::mpi_number;
     int omp_num = Parallel_Global::omp_number;
-#elif
+#else
     int mpi_num = 1;
     int omp_num = 1;
 #endif

--- a/source/module_io/para_json.cpp
+++ b/source/module_io/para_json.cpp
@@ -31,7 +31,7 @@ void json_output()
 #ifdef __MPI
     if (GlobalV::MY_RANK == 0)
         AbacusJson::write_to_json("abacus.json");
-#elif
+#else
     AbacusJson::write_to_json("abacus.json");
 #endif
 #endif // __RAPIDJSON
@@ -51,7 +51,7 @@ void gen_stru_wrapper(UnitCell *ucell){
 #ifdef __MPI
     if (GlobalV::MY_RANK == 0)
         gen_stru(ucell);
-#elif
+#else
     gen_stru(ucell);
 #endif
 #endif

--- a/source/module_io/para_json.cpp
+++ b/source/module_io/para_json.cpp
@@ -1,12 +1,12 @@
 
 #include "para_json.h"
 
+#include "module_base/global_variable.h"
+
 #include <ctime>
 #include <iomanip>
 #include <sstream>
 #include <string>
-
-#include "module_base/global_variable.h"
 #ifdef __RAPIDJSON
 #include "json_output/abacusjson.h"
 #include "json_output/general_info.h"
@@ -37,7 +37,8 @@ void json_output()
 #endif // __RAPIDJSON
 }
 
-void create_Json(UnitCell *ucell,Input *input){
+void create_Json(UnitCell* ucell, Input* input)
+{
 #ifdef __RAPIDJSON
     gen_general_info(input);
     gen_init(ucell);
@@ -46,7 +47,8 @@ void create_Json(UnitCell *ucell,Input *input){
     json_output();
 }
 
-void gen_stru_wrapper(UnitCell *ucell){
+void gen_stru_wrapper(UnitCell* ucell)
+{
 #ifdef __RAPIDJSON
 #ifdef __MPI
     if (GlobalV::MY_RANK == 0)

